lib/src/main/java/com/basistheory/android/view/TextElement.kt

```diff
@@ -16,15 +16,17 @@ import com.basistheory.android.event.BlurEvent
 import com.basistheory.android.event.ChangeEvent
 import com.basistheory.android.event.ElementEventListeners
 import com.basistheory.android.event.FocusEvent
-import com.basistheory.android.view.mask.MaskWatcher
+import com.basistheory.android.model.KeyboardType
+import com.basistheory.android.model.InputAction
+import com.basistheory.android.view.mask.Mask
 
 open class TextElement : FrameLayout {
     private var attrs: AttributeSet? = null
     private var defStyleAttr: Int = androidx.appcompat.R.attr.editTextStyle
     private var input: AppCompatEditText = AppCompatEditText(context, attrs, defStyleAttr)
     private var defaultBackground = input.background // todo: find a better way to reference this
     private val eventListeners = ElementEventListeners()
-    private var maskWatcher: MaskWatcher? = null
+    private var maskValue: Mask? = null
```

Comment:
I wouldn't call this maskValue, I even think the name of the class itself is off because it is more of like a mask helper or enforcer

Reply:
or masker lol

Reply:
I can't think of a better name ATM, maybe we can change it with another PR